resilience4j-spring-boot/src/main/java/io/github/resilience4j/bulkhead/autoconfigure/BulkheadAutoConfiguration.java

```diff
@@ -42,15 +42,19 @@
 @AutoConfigureBefore(EndpointAutoConfiguration.class)
 public class BulkheadAutoConfiguration {
 
-	@Bean
+	@Configuration
 	@ConditionalOnClass(value = {Endpoint.class})
-	public BulkheadEndpoint bulkheadEndpoint(BulkheadRegistry bulkheadRegistry, ThreadPoolBulkheadRegistry threadPoolBulkheadRegistry) {
-		return new BulkheadEndpoint(bulkheadRegistry, threadPoolBulkheadRegistry);
-	}
+	public static class BulkheadEndpointConfiguration {
```

Comment:
I wrapped Endpoint Bean with Configuration. In spring-boot1, it causes Error because of `AbstractEndpoint` which is used in custom endpoint class.